src/elements/FormTextInput/DropdownInput/DropdownInput.tsx

```diff
@@ -0,0 +1,70 @@
+import {
+	StyleProp, TextInput, TextInputProps, TextStyle,
+} from 'react-native';
+import React, { Ref } from 'react';
+import RNPickerSelect, { Item } from 'react-native-picker-select';
+import { Icon } from '@elements';
+import * as colors from '@util/colors';
+import defaultStyle from './DropdownInput.styles';
+
+
+interface DropdownInputProps extends TextInputProps {
+	/** User-submitted value. */
+	value: TextInputProps['value'];
+
+	/** Callback used with every keystroke within the input. */
+	setValue: TextInputProps['onChangeText'];
+
+	/** Styling to override default input styling. */
+	inputStyle?: StyleProp<TextStyle>;
+
+	/** Reference to the text input for programmatic manipulation. */
+	forwardedRef?: Ref<TextInput>;
+
+	/** Dropdown data for dropdownList */
+	dropdownData?: Array<string>;
+
+}
+
+const DropdownInput = (props: DropdownInputProps) => {
+	const {
+		style, value, setValue, dropdownData,
+	} = props;
+	const formattedData: Item[] = [];
+	dropdownData?.forEach(item => formattedData.push({ label: item, value: item }));
+	return (
+		<RNPickerSelect
+			style={
+				{
+					...defaultStyle,
+					iconContainer: {
+						top: 13,
+						right: 2,
+					},
+					placeholder: {
+						color: colors.NAVY_BLUE,
+						fontSize: 16,
+					},
+				}
+
+			}
+			placeholder={{
```

Comment:
The placeholder here creates another dropdown item, creating 2 `"WA"` selections. I believe `DonorRegistrationScreen` passes `"WA"` as an initial value and can be passed down to `DropdownInput`

Reply:
Thanks for reviewing! Are you testing on android? It works fine on IOS...

Reply:
Oh I found it out, nice catch! I'll update and push modification late.